tests/ui/regions/desktop/search.py

```diff
@@ -0,0 +1,32 @@
+from pypom import Region
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
```

Comment:
Found one more! :smiling_imp:

Reply:
Dang it!

Reply:
I'll fix, do a THOROUGH search for any remaining, squash, and push.